lib/config/global.ts

```diff
@@ -4,9 +4,9 @@ export class GlobalConfig {
   // TODO: once global config work is complete, add a test to make sure this list includes all options with globalOnly=true (#9603)
   private static readonly OPTIONS: (keyof RepoGlobalConfig)[] = [
     'allowCustomCrateRegistries',
-    'allowedPostUpgradeCommands',
+    'allowedPeriUpgradeCommands',
```

Comment:
```suggestion
    'allowedUpgradeCommands',
```

Reply:
I used the peri prefix so that there is no ambiguity. I wouldn't want users to think that it does not include upgrade commands allowed in the "allowedUpgradeCommands" property, such as `"allowedUpgradeCommands": ["pod install", "npm install", "mvn install"]`.

If you think there will be no risk of ambiguity, that's fine for me.